packages/neovim/build.sh

```diff
@@ -40,6 +62,10 @@ termux_step_pre_configure() {
 	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" -DLUA_MATH_LIBRARY=$TERMUX_STANDALONE_TOOLCHAIN/sysroot/usr/lib/$TERMUX_HOST_PLATFORM/$TERMUX_PKG_API_LEVEL/libm.so"
 }
 
+termux_step_configure() {
```

Comment:
Are you overriding termux_step_configure here to make it not run? 

Looks weird when termux_step_pre_configure is identical to termux_step_configure

Reply:
It's my merging error, will fix now.